app/components/primer/image.rb

```diff
@@ -27,7 +27,7 @@ class Image < Primer::Component
     # @param lazy [Boolean] Whether or not to lazily load the image.
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(src:, alt:, lazy: false, **system_arguments)
-      @system_arguments = system_arguments
+      @system_arguments = deny_single_argument(:tag, "This component has a fixed tag.", **system_arguments)
```

Comment:
I think we could still have a `deny_tag_argument` helper:

```rb
def deny_tag_argument(**arguments)
  deny_single_argument(:tag, "This component has a fixed tag.", **system_arguments)
end
```

since this will be used in many components